hasher-matcher-actioner/hmalib/common/config.py

```diff
@@ -310,7 +314,25 @@ def update_config(config: HMAConfig) -> None:
     get_dynamodb().meta.client.put_item(
         TableName=_TABLE_NAME,
         Item=_config_to_dynamodb_item(config),
+        ConditionExpression=Attr("ConfigType").not_exists(),
```

Comment:
Dumb question: Will this suffice? Shouldn't we also check whether the ConfigName `.not_exists()`?

Reply:
I strongly feel this will prevent creation of multiple configs of the same type.

Reply:
yeah this is a little tricky, there is one post to explain the way dynamoDB works
https://www.alexdebrie.com/posts/dynamodb-condition-expressions/
<img width="690" alt="Screen Shot 2021-04-29 at 5 17 14 PM" src="https://user-images.githubusercontent.com/81996660/116619677-c4b89700-a90e-11eb-90ad-50e211791826.png">
Basically when want to check duplicate, we just need to send PK, dynamoDB will automatically check both PK and SK

Reply:
Checked this out on my devcontainer. Seems to be working fine. The ConfigName check is implicit.